src/aap_eda/services/project/scm.py

```diff
@@ -243,7 +243,7 @@ def build_url(
     @classmethod
     def decrypt_key_file(cls, key_file: str, password: str) -> None:
         result = subprocess.run(
-            [KEYGEN_COMMAND, "-p", "-P", password, "-N", '""', "-f", key_file],
+            [KEYGEN_COMMAND, "-p", "-P", password, "-N", "", "-f", key_file],
```

Comment:
@bzwei `[-N new_passphrase] ` is an optional parameter, do you have to set it to an empty string?